CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+cmake_minimum_required(VERSION 3.16.0)
+project(openfx VERSION 1.4.0 LANGUAGES CXX)
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_STATIC_LIBRARY_PREFIX "lib")
+add_definitions(-D_HAS_AUTO_PTR_ETC)
+
+# Flags
+if(MSVC)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWINDOWS -DNOMINMAX")
+else()
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wno-deprecated -Wno-deprecated-declarations")
+endif()
+
+# Conan
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
```

Comment:
This seems to require conan; plain cmake fails here with "include could not find requested file" conanbuildinfo.cmake. Is that your intent here? I think it would be better to allow a plain cmake build if possible.